api/app/serializers/spree/v2/storefront/shipment_serializer.rb

```diff
@@ -0,0 +1,16 @@
+module Spree
+  module V2
+    module Storefront
+      class ShipmentSerializer < BaseSerializer
+        set_type :shipment
+
+        attributes :tracking, :number, :cost, :shipped_at, :state, :created_at, :updated_at,
```

Comment:
@VygovskySergey let's also add `display_*` methods eg. `display_promo_total`

also we shouldn't expose timestamps